2-mandatory/4-stocks.js

```diff
@@ -34,7 +34,18 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  let newArray = [];
+
+  for (stocks of closingPricesForAllStocks) {
+    let average = 0;
```

Comment:
You've called this variable `average` but looks like you're just using it to calculate the sum.